src/delegate.erl

```diff
@@ -200,6 +235,16 @@ group_pids_by_node(Pids) ->
                  node(Pid), fun (List) -> [Pid | List] end, [Pid], Remote)}
       end, {[], maps:new()}, Pids).
 
+group_local_call_pids_by_node(Pids) ->
+    {LocalPids0, Grouped0} = group_pids_by_node(Pids),
+    maps:fold(fun(K, V, {AccIn, MapsIn}) -> 
+        case V of
+            %% just one Pid for the node
+            [S_v] -> {[S_v | AccIn], MapsIn};
+            _ -> {AccIn, maps:update_with(K, fun(V1) -> V1 end, V, MapsIn)}
```

Comment:
I'm not sure why this update is necessary. If the value is a list of more than one pid, should the clause simply continue?

Reply:
_ -> {AccIn, maps:update_with(K, fun(V1) -> V1 end, V, MapsIn)}  ：  If the value is a list of more than one pid,  the (K,V) will be put to the new map which will be called through delegate.